fern/docs/pages/docs/writing-content.mdx

```diff
@@ -194,7 +194,7 @@ typography:
 
 ### Tabs
 
-Within the navigation you can add `tabs`. Tabs are used to group content together. Let's add a new tab so that we can include `Help Center` content in addition to our API Reference. Each tab has a `title` and `icon`. [Browse the icons available](./components/icons) from FontAwesome.
+Within the navigation you can add `tabs`. Tabs are used to group content together. Let's add a new tab so that we can include `Help Center` content in addition to our API Reference. Each tab has a `title` and `icon`. [Browse the icons available](https://docs.buildwithfern.com/generate-docs/component-library/icons) from FontAwesome.
```

Comment:
```suggestion
Within the navigation, you can add `tabs`. Tabs are used to group content together. Let's add a new tab so that we can include `Help Center` content in addition to our API Reference. Each tab has a `title` and `icon`. [Browse the icons available](/generate-docs/component-library/icons) from FontAwesome.
```